web/src/components/navigation/navbarpc.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Landing from '../../pages/landing';
+import Signin from '../../pages/signin';
+import Events from '../../pages/events';
+import Logo from '../../img/tinysocial-logo.png';
+import {
+  BrowserRouter,
+  Switch,
+  Route,
+  Link,
+} from 'react-router-dom';
+import {
+  AppBar,
+  Tabs,
+  Tab,
+  Grid,
+} from '@material-ui/core';
+
+function NavBarPC() {
+  return (
+      <BrowserRouter>
+        <div>
+          <Route
+              path="/"
+              render={({location}) => (
+                  <>
+                    <AppBar color="default">
+                      <Grid container direction="row" justify="space-between"
```

Comment:
Usually attribute are listed in lexicographical order (so it's easier to read and keep track)